client/coral-admin/src/routes/Moderation/components/Moderation.js

```diff
@@ -19,6 +20,17 @@ export default class Moderation extends Component {
   }
 
   componentWillMount() {
+    const {router, root: {settings, reportedCount}} = this.props;
+
+    // Will update the URL to premod or new based on the moderation of All Streams
+    // If there is a reported comments, reported queue has priority!
+    if (!router.params.id) {    
```

Comment:
The conditional should be to check if we are currently at the `IndexRoute` and redirect to the correct "queue" if true.